drivers/adc/ad463x/ad463x.c

```diff
@@ -161,7 +192,7 @@ int32_t ad463x_spi_reg_write_masked(struct ad463x_dev *dev,
 int32_t ad463x_set_pwr_mode(struct ad463x_dev *dev, uint8_t mode)
 {
 	if ((mode != AD463X_NORMAL_MODE) && (mode != AD463X_LOW_POWER_MODE))
-		return -1;
+		return -EINVAL;
```

Comment:
don't do these kind of changes in the same patch. this patch adds a new feature as stated in the commit title/body.